htdocs/core/class/commonobject.class.php

```diff
@@ -6571,6 +6571,7 @@ function setListDependencies() {
 								$("select[name=\""+parent_list+"\"]").change(function() {
 									showOptions(child_list, parent_list);
 								});
+								showOptions(child_list, parent_list);
```

Comment:
Just before, we already called showOptions(child_list, parent_list); but only wen necessary.
Adding this means in some cas, we will call it twice ?